src/shared/components/workbasket-filters/workbasket-filters.component.ts

```diff
@@ -279,4 +280,13 @@ export class WorkbasketFiltersComponent implements OnInit {
     this.selected.caseType = undefined; // option should be blank rather than "Select a value" in case of reset.
     this.selectedJurisdictionCaseTypes = null;
   }
+
+  private setFocusToTop() {
```

Comment:
@phillip-whitaker - more of a refactoring optimisation, is there anywhere you can put the `setFocusToTop()` function, which can be used by both components? This was also defined in `search-filters.component.ts`.

Reply:
I agree about the scrollTo function etc. It would be nice in a common function. However.. can I be cheeky and ask they we don't? Purely because it is an old change and would just like to get it through without revisiting everything it did etc.

Reply:
@phillip-whitaker - fair enough, it's a small function so there probably isn't that much to be gained. Maybe it could be refactored in future...